mne_bids/utils.py

```diff
@@ -726,6 +726,9 @@ def _read_events(events_data, event_id, raw, ext):
         The data as MNE-Python Raw object.
     ext : str
         The extension of the original data file.
+    verbose : bool | str | int | None
+        If not None, override default verbose level (see :func:`mne.verbose`
+        and :ref:`Logging documentation <tut_logging>` for more).
```

Comment:
```suggestion
        If not None, override default verbose level (see :func:`mne.verbose`).
```

the sphinx ref to the mne docs won't work in mne-bids, so I suggest to remove it.

Reply:
+1